app/classifier/tasks/dropdown/dropdown-dialog.cjsx

```diff
@@ -77,7 +85,7 @@ DropdownDialog = React.createClass
     @setState editSelect: select
 
   onClickAddOption: ->
-    @addOption(@refs.optionInput.value)
+    @addOption @refs.optionInput.value
     @refs.optionInput.value = ''
```

Comment:
should this also be updated to use state instead of refs? i'm fairly certain i know how to, but in my head it looks like more code so i'm not sure if this is a scenario where editing the ref/DOM directly is ok? editing state instead of the ref/DOM directly makes total sense though.